cmd/kops/get.go

```diff
@@ -46,9 +46,15 @@ var (
 	# Get a cluster
 	kops get cluster k8s-cluster.example.com
 
-	# Get a cluster YAML cluster spec
+	# Get a cluster YAML cluster desired configuration
 	kops get cluster k8s-cluster.example.com -o yaml
 
+	# Get a cluster YAML cluster actual configuration
+	kops get cluster k8s-cluster.example.com --full -o yaml
+
+	# Save a cluster desired configuration to YAML file
+	kops get cluster k8s-cluster.example.com -o yaml > cluster-desired-config.yaml
```

Comment:
I removed the examples `get cluster` and `get ig` and expanded on just `get` instead.
(I don't think get `ig being` would be used too much now).